graph/db/sql_store.go

```diff
@@ -2404,9 +2391,19 @@ func (s *SQLStore) PruneGraph(spentOutputs []*wire.OutPoint,
 			}
 
 			closedChans = append(closedChans, info)
+
```

Comment:
Do we also need to do a list query for the `DeleteChannel` call above? Probably not, as we don't expect thousands of channels to be closed by a single block worth of transactions...